Gopkg.toml

```diff
@@ -0,0 +1,26 @@
+
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/juju/errgo"
```

Comment:
I recon it's only external dependency that we rely on a branch instead of version (aka tag or sha).
We discussed we want to pin backoff as this is external. Shouldn't it apply here as well?

Reply:
Is backoff used in microerror?

Reply:
No :) The question is if `errgo` should be pinned.

Reply:
I don't think so because it is pinned in microerror.